src/main/java/com/fasterxml/jackson/databind/deser/BasicDeserializerFactory.java

```diff
@@ -879,10 +881,17 @@ protected void _addExplicitPropertyCreator(DeserializationContext ctxt,
                     */
                 }
                 name = candidate.findImplicitParamName(i);
-                _validateNamedPropertyParameter(ctxt, beanDesc, candidate, i,
+                // [databind#562] Allow @JsonAnySetter in creators. Introspection is done here only because of
+                // _validateNamedPropertyParameter() check. Otherwise, in constructCreatorProperty seems appropriate....
+                hasAnySetter = ctxt.getAnnotationIntrospector().hasAnySetter(param);
+                if (Boolean.TRUE.equals(hasAnySetter)) {
```

Comment:
Should we verify that only one any-setter is found here? (I assume multiple ones are not allowed as it would not be clear which one is to be used?)

Reply:
Make sense. Thank you for doing the job👍🏼